uint8 UNKNOW = 0
uint8 RAINY = 1
uint8 FOGGY = 2
uint8 SUNNY = 3
uint8 CLOUDY = 4

uint8 weather
uint8 road_type
